src/app/profile/email-verification/email-verification.spec.ts

```diff
@@ -0,0 +1,61 @@
+import { NO_ERRORS_SCHEMA } from '@angular/core';
+import { async, TestBed } from '@angular/core/testing';
+import { FormsModule } from '@angular/forms';
+import { HttpModule } from '@angular/http';
+import { By } from '@angular/platform-browser';
+
+import { EmailVerificationComponent } from './email-verification.component';
+
+describe('Email Verification Component', () => {
+  let fixture;
+  let email_true;
+  let email_false;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [FormsModule, HttpModule],
+      declarations: [EmailVerificationComponent],
+      providers: [],
+      // Tells the compiler not to error on unknown elements and attributes
+      schemas: [NO_ERRORS_SCHEMA]
+    });
+    email_true = {
+      attributes: {
+        verified: true,
+        message: '',
+        secMessage: ''
+      }
+    };
+    email_false = {
+      attributes: {
+        verified: false,
+        message: '',
+        secMessage: ''
+      }
+    };
+    fixture = TestBed.createComponent(EmailVerificationComponent);
+  });
+
+  it('should verify email is verified', async(() => {
+    let comp = fixture.componentInstance;
+    let debug = fixture.debugElement;
+    comp.email_true = email_true;
+    fixture.detectChanges();
+    let element = debug.queryAll();
+    fixture.whenStable().then(() => {
+      console.log(element.find('img').prop('src'));
+      expect(element.find('img').prop('src')).toEqual('../../../assets/images/trophy.png');
```

Comment:
You appear to be setting verified to a boolean value here, but testing a string value in the component and HTML template. 

For example, I see this in the HTML template:
[src]="verified==='true'

I believe Jenkins may be failing on the unit tests because of this boolean Vs string comparison? That is, you end up with neutralface.png instead of trophy.png.